app/helpers/peer_review_helper.rb

```diff
@@ -45,10 +43,13 @@ def create_map_group_id_to_name(reviewer_groups, reviewee_groups)
   # Returns a map of reviewer_id => num_of_reviews
   def create_map_number_of_reviews_for_reviewer(reviewer_groups)
     number_of_reviews_for_reviewer = {}
+
+    # TODO - [N+1 query] Optimize with: SELECT reviewer_id, COUNT(reviewer_id) FROM peer_reviews GROUP BY reviewer_id
```

Comment:
Bothers me that I didn't get this one, I figured I'd leave a comment and come back to it in the future, or maybe implement it in raw SQL and then put a nice TODO next to it to be fixed later. I tried a lot of things to attempt to make this work, and rails fought me too hard... I'm not giving up on this but rather putting it as a 'to do later', though if you know a way to get the above query to work that would be awesome.